<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Konfig_uzytkownika</class>
 <widget class="QDialog" name="Konfig_uzytkownika">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konfiguracja użytkowników</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_szukaj">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_Wyszukaj">
        <property name="text">
         <string>szukaj</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_wedlug">
        <property name="text">
         <string>według</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_szukaj">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>idp</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>imie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nazwisko</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>login</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_szukaj"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_haslo">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_nowe_haslo">
        <property name="text">
         <string>Nowe hasło</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_nowe_haslo">
        <property name="inputMethodHints">
         <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_potw_haslo">
        <property name="text">
         <string>Potwierdź hasło</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_potw_haslo">
        <property name="inputMethodHints">
         <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_OK">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="groupBox_user">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_idp">
        <property name="text">
         <string>idp</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_idp"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_imie">
        <property name="text">
         <string>Imię</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_imie"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_nazwisko">
        <property name="text">
         <string>Nazwisko</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_nazwisko"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_login">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_login"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBox_blokada">
        <item>
         <property name="text">
          <string>TAK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NIE</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_blokada">
        <property name="text">
         <string>blokada</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBox_admin">
        <item>
         <property name="text">
          <string>TAK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NIE</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_admin">
        <property name="text">
         <string>admin</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_przyciski">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_zmienhaslo">
        <property name="text">
         <string>zmień hasło</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Edytuj">
        <property name="text">
         <string>Edytuj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Dodaj">
        <property name="text">
         <string>Dodaj</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTableView" name="tableView_uzytkownicy"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
